guides/source/getting_started.md

```diff
@@ -1991,6 +1991,140 @@ the `app/views/comments` directory.
 The `@article` object is available to any partials rendered in the view because
 we defined it as an instance variable.
 
+
+### Using Concerns
+
+  A concern is any module that contains methods you would like to be able to share across multiple controllers or models. In other frameworks they are often known as mixins. They are a good way to keep your controllers and models small and keep your code DRY.
```

Comment:
While this is accurate, concerns are also used more broadly for "extracting a slice of model that doesn’t seem part of its essence", to quote [DHH](https://signalvnoise.com/posts/3372-put-chubby-models-on-a-diet-with-concerns). Given that vision, we may not want to suggest in the guide that concerns are _only_ for DRYing up code.

Reply:
@dzunk I agree, but I'm not sure how to put that in a way that doesn't lead to more questions (in the article, DHH says " (what is and isn’t in the essence of a model is a fuzzy line and a longer discussion)" ) . 

perhaps something like "They are a good way to keep your controllers and models small and keep your code DRY, as well as extract any generic attributes?" 

Or perhaps I should remove this line entirely, and let the user infer when this might be useful? I'm not so sure about that. 🤔

Reply:
This is a tough one to describe in a beginner friendly way :|

I'd describe concerns in Rails as an approach to make large models easier to understand and manage by extracting logic into well defined slices of responsibility. This comes with the benefit of reusability when multiple models share the same concerns.

Reply:
This is a tough one to describe in a beginner friendly way :|

I'd describe concerns in Rails as an approach to make large models easier to understand and manage by extracting logic into well defined slices of responsibility. This comes with the benefit of reusability when multiple models share the same concerns.

Reply:
@fylooi Thanks for the help here! I've updated the guide with your suggestion, as well as amended a paragraph further down (which I will tag you on) . Would be great to hear your thoughts!